icekit/admin_tools/templates/admin/base_site.html

```diff
@@ -18,18 +18,18 @@
 	{% compress css %}
 		{# Bootstrap, then default admin styles, then our styles. #}
 		{# This avoids overriding the admin with Bootstrap, and allows us to override anything. #}
-		<link rel="stylesheet" href="{% static 'bootstrap/dist/css/bootstrap.css' %}" />
+		<link rel="stylesheet" href="{% static 'icekit/bower_components/bootstrap/dist/css/bootstrap.css' %}" />
 		<link rel="stylesheet" href="{% static 'admin/css/base.css' %}" />
 		<link type="text/less" rel="stylesheet" href="{% static 'admin/css/icekit_dashboard.less' %}" />
 		<link type="text/less" rel="stylesheet" href="{% static 'admin/css/override.less' %}" />
 	{% endcompress %}
-	<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css">
+	<link rel="stylesheet" href="{% static 'icekit/bower_components/font-awesome/css/font-awesome.css' %}">
```

Comment:
This is an incidental fix that I've been meaning to make for at least a year. 

There's no benefit to using a CDN for the admin, and it adds another point of failure.

My guess is that it was added in by Pat during initial prototyping and no one has never corrected it.